connector/src/main/java/org/geysermc/connector/network/translators/item/RecipeRegistry.java

```diff
@@ -46,22 +45,44 @@
      * A list of all possible leather armor dyeing recipes.
      * Created manually.
      */
-    public static List<CraftingData> LEATHER_DYEING_RECIPES = new ObjectArrayList<>();
+    public static final List<CraftingData> LEATHER_DYEING_RECIPES = new ObjectArrayList<>();
     /**
      * A list of all possible firework rocket recipes, including the base rocket.
      * Obtained from a ProxyPass dump of protocol v407
      */
-    public static List<CraftingData> FIREWORK_ROCKET_RECIPES = new ObjectArrayList<>(21);
+    public static final List<CraftingData> FIREWORK_ROCKET_RECIPES = new ObjectArrayList<>();
     /**
      * A list of all possible firework star recipes.
      * Obtained from a ProxyPass dump of protocol v407
      */
-    public static List<CraftingData> FIREWORK_STAR_RECIPES = new ObjectArrayList<>(40);
+    public static final List<CraftingData> FIREWORK_STAR_RECIPES = new ObjectArrayList<>();
     /**
      * A list of all possible shulker box dyeing options.
      * Obtained from a ProxyPass dump of protocol v407
      */
-    public static List<CraftingData> SHULKER_BOX_DYEING_RECIPES = new ObjectArrayList<>();
+    public static final List<CraftingData> SHULKER_BOX_DYEING_RECIPES = new ObjectArrayList<>();
+    /**
+     * A list of all possible suspicious stew recipes.
+     * Obtained from a ProxyPass dump of protocol v407
+     */
+    public static final List<CraftingData> SUSPICIOUS_STEW_RECIPES = new ObjectArrayList<>();
+    /**
+     * A list of all possible tipped arrow recipes.
+     * Obtained from a ProxyPass dump of protocol v407
+     */
+    public static final List<CraftingData> TIPPED_ARROW_RECIPES = new ObjectArrayList<>();
+
+    // Unknown UUIDs: 442d85ed-8272-4543-a6f1-418f90ded05d 8b36268c-1829-483c-a0f1-993b7156a8f2 602234e4-cac1-4353-8bb7-b1ebff70024b 98c84b38-1085-46bd-b1ce-dd38c159e6cc
+    // d81aaeaf-e172-4440-9225-868df030d27b b5c5d105-75a2-4076-af2b-923ea2bf4bf0 00000000-0000-0000-0000-000000000002 85939755-ba10-4d9d-a4cc-efb7a8e943c4
+    // d392b075-4ba1-40ae-8789-af868d56f6ce aecd2294-4b94-434b-8667-4499bb2c9327
```

Comment:
Are these just left in from development or do they have any use? If they have a reason to be here can they be commented accordingly.

Reply:
Ideally they're here as a reference to whoever in the future wants to tackle the remaining missing recipes. Of course as it's now in git commits I can delete this off from the final product.